### PR TITLE
Improve feedback to user when objects aren't retrieved due to RBAC 

### PR DESCRIPTION
**Pull request by AlinaGoaga**
Closes #[2427](https://github.com/weaveworks/weave-gitops-enterprise/issues/2747)

**What changed?**
We're doing a pilot on the Application table to display an info note containing the clusters and namespaces that have been searched to retrieved the data. 

**Why was this change made?**
Sometimes users don't understand why they cannot see certain objects in the UI. This can be caused by a lack of permissions. We're looking to include more information on which clusters and namespaces have been searched trying to retrieve objects for a specific page. 

**How was this change implemented?**
`ListObjects` now returns in addition to the Objects and associated Errors, a list of searched Namespaces (including cluster name and an array of namespaces per cluster)

Screenshot 2023-05-08 at 18 48 40

### CODE REVIEW

**Comment by bigkevmcd** [comment]
@foot Another one for you when you get back :-)

**Comment by bigkevmcd** [comment]
@foot Another one for you when you get back :-)

File: ui/hooks/automations.ts [hunk 1]
```diff
@@ -52,6 +64,21 @@ export function useListAutomations(
...
+              final.searchedNamespaces.push({
+                [k as string]: response.searchedNamespaces[k].namespaces,
+              });
+            } else {
+              final.searchedNamespaces[existingKeys.indexOf(k)][k] = _.uniq([
+                ...final.searchedNamespaces[existingKeys.indexOf(k)][k],
+                ...response.searchedNamespaces[k].namespaces,
+              ]);
+            }
+          }
```
Comment (foot) [inline ui/hooks/automations.ts:81]: Pull this out into a helper and add a couple of small tests?

**Comment by foot** [comment]
Tests are hanging 🤔

**Review by foot — APPROVED** [review]

**Pull request merged**